### PR TITLE
[STABLE] Set custom SG

### PR DESCRIPTION
**Pull request by mikkeloscar**
Cherry-pick to stable of: https://github.com/zalando-incubator/kubernetes-on-aws/pull/3780

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
👍

**Comment by hjacobs** [comment]
This PR rolls out kube-ingress-aws-controller [v0.11.10](https://github.com/zalando-incubator/kube-ingress-aws-controller/releases/tag/v0.11.10) with the fix https://github.com/zalando-incubator/kube-ingress-aws-controller/pull/381

File: cluster/manifests/ingress-controller/deployment.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: kube-system
   labels:
     application: kube-ingress-aws-controller
-    version: v0.11.7
```
Comment (hjacobs) [inline cluster/manifests/ingress-controller/deployment.yaml:8]: What are other changes between 0.11.7 and 0.11.10?

Comment (gargravarr) [reply cluster/manifests/ingress-controller/deployment.yaml:8]: [v0.11.9](https://github.com/zalando-incubator/kube-ingress-aws-controller/releases/tag/v0.11.9) is a noop (fixes Github actions)
[v0.11.8](https://github.com/zalando-incubator/kube-ingress-aws-controller/releases/tag/v0.11.8) are dependency updates reported by dependabot.

**Comment by gargravarr** [comment]
:+1:

**Pull request merged**